dbt_project.yml

```diff
@@ -0,0 +1,12 @@
+# Compile configuration
+source-paths: ["model"]   # paths with source code to compile
```

Comment:
i'm not sure that we want to check this in. If we do, we probably want to change `"model"` to `"models"`